Build failure on `uct_ib_log_dump_sg_list`

### Describe the bug
Build fails:

```cpp
  CC       ud/accel/libuct_ib_la-ud_mlx5.lo
/datasets/pentschev/src/ucx-master/contrib/../src/uct/ib/base/ib_log.c: In function ‘uct_ib_dump_exp_send_wr’:
/datasets/pentschev/src/ucx-master/contrib/../src/uct/ib/base/ib_log.c:330:42: error: passing argument 7 of ‘uct_ib_log_dump_sg_list’ makes integer from pointer without a cast [-Werror=int-conversion]
                            data_dump_cb, s, ends - s);
                                          ^
/datasets/pentschev/src/ucx-master/contrib/../src/uct/ib/base/ib_log.c:42:6: note: expected ‘int’ but argument is of type ‘char *’
 void uct_ib_log_dump_sg_list(uct_ib_iface_t *iface, uct_am_trace_type_t type,
      ^~~~~~~~~~~~~~~~~~~~~~~
/datasets/pentschev/src/ucx-master/contrib/../src/uct/ib/base/ib_log.c:330:45: error: passing argument 8 of ‘uct_ib_log_dump_sg_list’ makes pointer from integer without a cast [-Werror=int-conversion]
                            data_dump_cb, s, ends - s);
                                             ^~~~
/datasets/pentschev/src/ucx-master/contrib/../src/uct/ib/base/ib_log.c:42:6: note: expected ‘char *’ but argument is of type ‘long int’
 void uct_ib_log_dump_sg_list(uct_ib_iface_t *iface, uct_am_trace_type_t type,
      ^~~~~~~~~~~~~~~~~~~~~~~
/datasets/pentschev/src/ucx-master/contrib/../src/uct/ib/base/ib_log.c:327:4: error: too few arguments to function ‘uct_ib_log_dump_sg_list’
    uct_ib_log_dump_sg_list(iface, UCT_AM_TRACE_TYPE_SEND, wr->sg_list,
    ^~~~~~~~~~~~~~~~~~~~~~~
/datasets/pentschev/src/ucx-master/contrib/../src/uct/ib/base/ib_log.c:42:6: note: declared here
 void uct_ib_log_dump_sg_list(uct_ib_iface_t *iface, uct_am_trace_type_t type,
      ^~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
```

It seems that this was introduced today and the parameter `data_dump_sge` is missing in the call to `uct_ib_log_dump_sg_list` above.

### Steps to Reproduce
- `../contrib/configure-devel --with-valgrind=no && make -j`
- master branch on https://github.com/openucx/ucx/commit/ad463435c014f18e98be04e2f4d54f27d15cf0f8

### Setup and versions
- OFED 4.7-3.2.9

--- a/src/uct/ib/base/ib_log.c
+++ b/src/uct/ib/base/ib_log.c
@@ -325,9 +325,9 @@ static void uct_ib_dump_exp_send_wr(uct_ib_iface_t *iface, struct ibv_qp *qp,
 #endif
 
    uct_ib_log_dump_sg_list(iface, UCT_AM_TRACE_TYPE_SEND, wr->sg_list,
-                           ucs_min(wr->num_sge, max_sge),
+                           wr->num_sge,
                            (wr->exp_send_flags & IBV_EXP_SEND_INLINE) ? -1 : 0,
-                           data_dump_cb, s, ends - s);
+                           data_dump_cb, max_sge, s, ends - s);
 }
 
 void __uct_ib_log_exp_post_send(const char *file, int line, const char *function,

